packages/yarnpkg-fslib/sources/xfs.ts

```diff
@@ -44,19 +43,24 @@ function registerCleanExit() {
   });
 }
 
+let tmpdir: PortablePath | null = null;
+let realTmpdir: PortablePath | null = null;
```

Comment:
Convert these to const?
```suggestion
const tmpdir = npath.toPortablePath(os.tmpdir());
const realTmpdir = new NodeFS().realpathSync(tmpdir);
```

This way lines 56-57 and 96-97 can be deleted.